.buildkite/scripts/build_packer_ci.sh

```diff
@@ -9,24 +9,22 @@
 set -euo pipefail
 
 source .buildkite/scripts/lib/packer.sh
-source .buildkite/scripts/lib/packer_constants.sh
 
-echo -e "--- :packer: Performing build of AMI"
 export GIT_SHA="${BUILDKITE_COMMIT}"
 export GIT_BRANCH="${BUILDKITE_BRANCH}"
+# This syntax marks these two as required
 : "${BUILDKITE_BUILD_NUMBER}"
```

Comment:
The `:` is the "command that does nothing" ([reference](https://www.gnu.org/savannah-checkouts/gnu/bash/manual/bash.html#Bourne-Shell-Builtins))

Using it in this construction (and with our `set -u` configuration), will fail the script if that variable is not set, since `:` expands the variable for us.

This is a convenient way to front-load the environment variable checking, particularly since the variable isn't actually used anywhere else in this script. (In fact, it's optional inside the Packer template, too, so unless we did something like this, we could just silently be missing it and not know until we went to go search for our AMIs, expecting them to be tagged with a build number.)